source/_patterns/01-molecules/content-header/content-header.twig

```diff
@@ -0,0 +1,24 @@
+<div class="content-header"{% include 'atoms-attributes' %}>
+
+  {% block body %}
+
+    <div class="content-header__body">
+
+      {% with contentTitle only %}
+
+        {% block title %}
+
+          {% set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['content-header__title']) }) %}
```

Comment:
will `attributes` ever be set here, after the `{% with contentTitle only %}` block?

Reply:
Yeah, if it's added to https://github.com/libero/pattern-library/blob/ccf438fe7ae51cdebac7b487718923bf5cbe18dc/source/_patterns/01-molecules/content-header/content-header.yaml#L4-L5 for example.

Reply:
Ahh so `with` is changing the scope, not filtering. Makes sense now